act-rules-format.bs

```diff
@@ -245,196 +267,224 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
   </ol>
 </div>
 
-An atomic rule expectation MUST only use information available in the [test aspects](#input-aspects), from the applicability, and other expectations of the same rule. No other information can be used in the expectation. So for instance, an expectation could be "Expectation 1 is true and ...", but it can't be "Rule XYZ passed and ...". This ensures the rule is encapsulated.
-
-
-Expectations for Composite Rules {#expectations-composite}
---------------------------------------------------------
+<div class="note">
+  <p>**Note:** Rules can be used to do complex aggregation by describing the logic in the expectations. e.g. "The page has a text alternative for 80% of all img elements".</p>
+</div>
 
-A [=composite rule=] MUST contain one or more expectations that describes the logic that is used to determine a single `passed` or `failed` [=outcome=] for each [=test target=], based on the outcomes of [=atomic rules=] or [=composite rules=] listed in its [input rules list](#input-rules-list).
+### Expectations for Composite Rules ### {#expectations-composite}
 
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations is false, the test target `failed` the rule. This works the same way for atomic rules. A composite rule expectation MUST NOT use information from [test aspects](#input-aspects).
+All expectations of a [=composite rule=] MUST describe the logic that is used to determine a single `passed` or `failed` [=outcome=] for a [=test target=], based on the [=outcomes=] of rules in its [input rules](#input-rules). A composite rule expectation MUST NOT use information from [input aspects](#input-aspects). The outcome for a composite rule is `inapplicable` when all input rules are inapplicable. 
 
 <div class="example">
-  <p>A composite rule for <a href="https://www.w3.org/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded" target="_blank">WCAG 2.1 Audio Description or Media Alternative</a> applies a logic across results from three atomic rules. The expectation of the composite rule is as follows:</p>
-  <blockquote>
-    <p>For each test target, the outcome of one of the following rules is `passed`:</p>
-    <ul>
-      <li>Video elements have an audio description (video-audio-desc)</li>
-      <li>Video elements have a media alternative (video-text-alt)</li>
-    </ul>
-  </blockquote>
+  <p>Composite rule: video elements have an audio description or media alternative ([WCAG 2.1, success criterion 1.2.3 Audio Description or Media Alternative](https://www.w3.org/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded)).</p>
+  <p>Each HTML `video` element meets all expectations from at least one of the following rules:</p>
+  <ul>
+    <li>video elements have a transcript</li>
+    <li>video elements have an audio description</li>
+    <li>video elements have a description track</li>
```

Comment:
In composite rules, is it required to include the Rule Identifiers of the atomic rules?